tests/capi/unittest_capi_inference.cc

```diff
@@ -2245,19 +2245,45 @@ TEST (nnstreamer_capi_util, availability_fail_06_n)
 TEST (nnstreamer_capi_util, element_available_01_p)
 {
   bool available;
-  int status;
+  int status, n_elems,  i;
+  /**
+   * If the element restriction list of nnstreamers is changed, this should also be changed.
```

Comment:
How about "If the allowed element list of nnstreamer ..." ?

Reply:
Thanks. Updated.